docs/configuration.md

```diff
@@ -18,6 +18,8 @@
   - `enable_unsafe_install` - set to `true` to enable the use of the `--git-url` and `--zip-file` flags with
     [`arduino-cli lib install`][arduino cli lib install]. These are considered "unsafe" installation methods because
     they allow installing files that have not passed through the Library Manager submission process.
+- `locale` - the language used by Arduino CLI, the parameter is the language identifier in the standard format
+  `<language>_<COUNTRY>.<encoding>` (for example `it` or `it_IT`, or `it_IT.UTF-8`).
```

Comment:
Is there an ISO (or other) standard for the locale codes used in this project? If so, I would recommend referencing that here instead of the fairly ambiguous description currently provided.

Reply:
Looking here https://en.wikipedia.org/wiki/Locale_(computer_software) it seems that the current POSIX identifier is defined by [ISO/IEC 15897](https://en.wikipedia.org/wiki/ISO/IEC_15897) but only the draft is available as a free download, the final document is available only with payment.

Maybe we can just say POSIX/Unix?